src/bunit.web/Rendering/Internal/Htmlizer.cs

```diff
@@ -222,6 +224,13 @@ private static int RenderChildren(HtmlRenderingContext context, ArrayRange<Rende
                 {
                     case bool flag when flag:
                         result.Add(" ");
+                        if (frame.AttributeName.StartsWith(BLAZOR_INTERNAL_ATTR_PREFIX, StringComparison.Ordinal))
+                        {
+                            // NOTE: this was added to make it more obvious
+                            //       that this is a generated/special blazor attribute
+                            //		 for internal usage
+                            result.Add(BLAZOR_ATTR_PREFIX);
+                        }
                         result.Add(frame.AttributeName);
                         break;
```

Comment:
Let's split this out into its own case, to keep the "original" blazor code separate, e.g.:

```suggestion
                    case bool flag when flag && frame.AttributeName.StartsWith(BLAZOR_INTERNAL_ATTR_PREFIX, StringComparison.Ordinal):
                        // NOTE: This was added to make it more obvious
                        // that this is a generated/special blazor attribute
                        //	for internal usage                    
                        result.Add(" ");
                        result.Add(BLAZOR_ATTR_PREFIX);
                        result.Add(frame.AttributeName);
                        break;                    
                    case bool flag when flag:
                        result.Add(" ");
                        result.Add(frame.AttributeName);
                        break;
```

Reply:
fine with me